packages/amplify-cli-core/src/state-manager/stateManager.ts

```diff
@@ -75,6 +76,35 @@ export class StateManager {
     return this.getData<$TSTeamProviderInfo>(filePath, mergedOptions);
   };
 
+  getCustomPolicies = (service: string, categoryName: string, resourceName: string): any => {
+    if (service != 'Lambda' && service != 'ElasticContainer') {
```

Comment:
super nit: for some reason `if (!(service === 'Lambda' || service === 'ElasticContainer'))` reads better to me. But it's the same logic, so this is probably just personal preference